tfmigrate/multi_state_migrator.go

```diff
@@ -15,6 +15,10 @@ type MultiStateMigratorConfig struct {
 	FromDir string `hcl:"from_dir"`
 	// ToDir is a working directory where states of resources move to.
 	ToDir string `hcl:"to_dir"`
+	// FromWorkspace is a workspace within FromDir
+	FromWorkspace string `hcl:"from_workspace,optional"`
+	// ToWorkspace is a workspace within ToDir
+	ToWorkspace string `hcl:"to_workspace,optional"`
```

Comment:
Could you add new attributes to the README?
https://github.com/minamijoyo/tfmigrate#migration-block-multi_state

Reply:
Added. I have also added a table of content as the Readme file is quite long.

Reply:
Thank you for the table of content. Looks pretty 👍